docs/web3.pm.rst

```diff
@@ -1,41 +1,53 @@
 Package Manager API
 ===================
 
-.. py:module:: web3.pm
-.. py:class:: PM
 
 The ``web3.pm`` object exposes methods to interact with Packages as defined by `ERC 1123 <https://github.com/ethereum/EIPs/issues/1123>`_.
 
+To learn more about the EthPM spec, visit the `documentation <http://ethpm.github.io/ethpm-spec/>`__.
+To learn more about the Py-EthPM library used in this module, visit the `documentation <https://py-ethpm.readthedocs.io/en/latest/>`__.
 
-Installation
-------------
 
-.. warning:: The PM module is still under development, and not all use-cases are currently supported, so it is not included by default in the web3 instance.
+.. WARNING::
 
-You must install the eth-pm module separately, until it is stable. Install with:
+   The ``web3.pm`` API is still under development and likely to change quickly.
 
-.. code-block:: python
+   Now is a great time to get familiar with the API, and test out writing
+   code that uses some of the great upcoming features.
 
-  pip install --upgrade ethpm
+   By default, access to this module has been turned off in the stable version of Web3.py:
 
-Attaching
----------
+   .. code-block:: python
 
-To use ``web3.pm``, attach it to your ``web3`` instance.
+      >>> from web3.auto import w3
+      >>> w3.pm
+      ...
+      AttributeError: The Package Management feature is disabled by default ...
 
-.. code-block:: python
+   In order to access these features, you can turn it on with...
 
-   from web3.pm import PM
-   PM.attach(web3, 'pm')
+   .. code-block:: python
+
+      >>> web3.enable_unstable_package_management_api()
+      >>> w3.pm
+      <web3.pm.PM at 0x....>
 
 
 Methods
 -------
+The following methods are available on the ``web3.pm`` namespace.
+
+.. autoclass:: web3.pm.PM
+   :members:
+
+.. note:: If you want to implement your own registry and use it with ``web3.pm``, you must create a subclass that inherits from ``ERCRegistry``, implements all the methods defined in ``ERCRegistry``, and manually set it as the ```registry`` attribute on ``web3.pm``.
```

Comment:
This seems like something that would be worth putting in it's own section and 1) explaining *why* someone would want to do this and 2) how they would do it with a code example.

If the code example isn't something you want to maintain or you don't see it as very useful then maybe just state that replacing the registry is beyond the scope of this guide.